Cargo.toml

```diff
@@ -29,6 +29,7 @@ members = [
     "libs/deer/macros",
     "libs/error-stack",
```

Comment:
I guess we want to exclude `experimental` in the `error-stack` `Cargo.toml`?

Reply:
done in [948d066](https://github.com/hashintel/hash/pull/5054/commits/948d0665e2ac0c154fe870199dff4e41ebc7f471)